recipe/meta.yaml

```diff
@@ -46,6 +46,7 @@ test:
     {% endfor %}
     - test -f ${PREFIX}/include/bzlib.h  # [unix]
     - test -f ${PREFIX}/lib/libbz2.a  # [unix]
+    - test -f ${PREFIX}/lib/libbz2.so.1.0  # [unix]
```

Comment:
The Makefile generates only the file `libbz2.so.1.0.6` and a symlink `libbz2.so.1.0 → libbz2.so.1.0.6`. Is an unversion symlink also needed?